modules/programs/vim-vint.nix

```
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.vim-vint;
+
+  yamlFormat = pkgs.formats.yaml { };
+
+in {
+  meta.maintainers = [ maintainers.tomodachi94 ];
+
+  options = {
+    programs.vim-vint = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        defaultText = "false";
+        description = ''
+          Whether to enable the Vint linter for Vimscript.
+        '';
+      };
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.vim-vint;
+        defaultText = literalExpression "pkgs.vim-vint";
+        example =
+          literalExpression "(pkgs.vim-vint.override { enableCheck = true; })";
+        description = ''
+          The <literal>vim-vint</literal> package to use.
+        '';
+      };
```

```suggestion
      package = mkPackageOption pkgs "vim-vint" { };
```

I don't think the example is very useful or specific to vim-vint?